rfcs/interop_2023.md

```diff
@@ -0,0 +1,100 @@
+- December 1 to December 16:
+  - Start review of tests. Write needed tests.
+  - Start involving PR teams. Write rough drafts of announcements. Begin creating dashboard.
+
+During 2023:
+
+- January 9 to January 20:
+  - Finalize test selection.
+  - Complete announcement plans. Prepare dashboard for launch.
+- January 23 to January 27: Launch.
+
+If necessary, additional changes to the test lists can be made throughout the year with the consensus of the team, see [updating the tests](#updating-the-tests).
+
+### Governance
+
+The [Interop team](https://github.com/orgs/web-platform-tests/teams/interop) will manage the overall Interop 2023 effort. The team will be formed by renaming the [’22 team](https://github.com/orgs/web-platform-tests/teams/interop-2022).
+
+The team will meet regularly, with agenda issues created in the repo at least 48 hours in advance, and meeting minutes posted in the issue.
+
+The team makes decisions based on consensus, which is defined as support from at least two participating organizations and no opposition.
+
+The Interop 2023 effort is subject to the [WPT code of conduct](https://github.com/web-platform-tests/wpt/blob/master/CODE_OF_CONDUCT.md).
+
+### Proposals
+
+During the proposals period (September 15 to October 15), proposals are submitted as issues in the [Interop repo](https://github.com/web-platform-tests/interop). A proposal should be as specific as reasonably possible, but multiple proposals may be grouped later in the selection process.
+
+There are two kinds of proposals: focus areas where progress is measured using test results, and investigation efforts where progress is measuring against a task list. For both kinds of proposals, finishing them should meaningfully advance the state of interoperability for the web platform.
+
+#### Focus areas
+
+These proposals are for existing web platform features with good a spec and test suite which can be used to improve interoperability. Progress is measured by test results on wpt.fyi.
+
+The following information should be provided in a proposal:
+
+- Is this area a problem for web developers? For example, survey data and browser bug counts can help.
+- Is this area a problem for users?
+- How widely used is this feature? For example, [use counters](https://www.chromestatus.com/metrics/feature/popularity), [HTTP Archive](https://httparchive.org/) and [State of CSS](https://2021.stateofcss.com/en-US/features/) can help.
+- Standard/specification for this feature.
+- A list of tests on wpt.fyi which will be used to score progress towards interoperability on the feature.
+
+#### Investigation efforts
+
+If an area doesn't have a good spec and test suite it's not possible to use wpt.fyi test results to judge progress. Instead, some other work is needed to bring the area to state where test results can be used.
+
+The following information should be provided in a proposal:
```

Comment:
This list is essentially duplicating the above. Do we just want to duplicate this, or move this up to the general "Proposals" section above?

Reply:
I just duplicated it because it was less work, and I think still easy enough to understand this way. I don't feel strongly about it though, feel free to push an edit. (A suggestion would span many lines, I presume.)

Reply:
I'll leave them duplicated.